### PR TITLE
Api 1996 saml proxy linting

### PR DESCRIPTION
**Pull request by lukemarkwordtlibertyits**
## Description
Added Linting to the `saml-proxy`

## For Consideration
Part of Typescript linting enforces good OOP practices. To fully enforce this a refactoring out of the scope of this PR would be required. Gradually improvements should be made to the Typescript files as deemed necessary.

## Tests
[regression tests](https://github.com/department-of-veterans-affairs/vets-saml-proxy/files/5151614/API1996_regressionTests.docx)

### CODE REVIEW

File: saml-proxy/src/routes/index.js [hunk 1]
```diff
@@ -4,82 +4,100 @@ import bodyParser from "body-parser";
...
 import createPassport from "./passport";
 import addRoutes from "./routes";
 import configureHandlebars from "./handlebars";
 import { getParticipant } from "./handlers";
 
-import promBundle from 'express-prom-bundle';
-import * as Sentry from '@sentry/node';
+import promBundle from "express-prom-bundle";
+import * as Sentry from "@sentry/node";
+// import { logger } from "handlebars";
```
Comment (beaugrantham-va) [inline saml-proxy/src/routes/index.js:26]: I'm seeing some things that were changed in previous PRs... can you pull in the latest master branch?

File: saml-proxy/src/SPConfig.js [hunk 2]
```diff
@@ -23,8 +26,7 @@ export default class SPConfig {
     this.idpThumbprint = argv.spIdpThumbprint;
     this.idpMetaUrl = argv.spIdpMetaUrl;
     this.audience = argv.spAudience;
-    this.providerName = argv.spProviderName,
-    this.acsUrls = argv.spAcsUrls;
+    (this.providerName = argv.spProviderName), (this.acsUrls = argv.spAcsUrls);
```
Comment (beaugrantham-va) [inline saml-proxy/src/SPConfig.js:29]: I don't get this change...

Comment (lukemarkwordtlibertyits) [reply saml-proxy/src/SPConfig.js:29]: may have been a rebasing thing. I dont like the way it looks. I will take it out.

File: saml-proxy/src/routes/index.js (hunk 1 above)
Comment (lukemarkwordtlibertyits) [reply saml-proxy/src/routes/index.js:26]: It was pulled. The rebasing was pretty big. I am going to have to go over these files again.

**Comment by beaugrantham-va** [comment]
As discussed offline, this PR is too big to thoroughly review. It should be broken up by rule (formatting-only changes followed by gradual introduction of code/logic changes).

**Pull request closed without merging**